src/legacy/ui/public/vis/editors/default/_sidebar.scss

```diff
@@ -225,3 +219,19 @@
   margin-top: $euiSizeS;
   margin-bottom: $euiSizeS;
 }
+
+.visEditorSidebar__aggGroup .euiAccordion {
+  // to apply truncated string with ellipsis for agg description
+  & > .euiFlexGroup > .euiFlexItem:first-child,
+  .euiAccordion__buttonContent,
+  // prevent appearing scroll bar in .euiAccordion__childWrapper due to negative margin on some children's EuiFlexGroup
+  .euiForm > .euiFormRow:first-child,
+  [aria-controls="advancedAccordion"] {
+    overflow: hidden;
+  }
+
+  // prevent hiding a part of the border of accordion caret icon on focus
+  & > .euiFlexGroup > .euiFlexItem:first-child > button {
+    padding: 0 0 1px 2px;
```

Comment:
Some paddings for caret icon border

Before | After 
----|----
<img width="302" alt="btnfocus bre" src="https://user-images.githubusercontent.com/11524637/61465989-cdcab500-a981-11e9-9446-7a472657bf69.png">|<img width="291" alt="btnfocus" src="https://user-images.githubusercontent.com/11524637/61465999-d28f6900-a981-11e9-8561-e68b21094093.png">